test/common.py

```diff
@@ -17,6 +17,7 @@
 import itertools
 import json
 import logging
+import multiprocessing
 import threading
```

Comment:
This line needs to be removed. Looks like I removed the wrong line. Confused how my pr passed though...